### ISSUE TITLE
SMP Alchemical Chemistry Set, RFTools and horrible lag

### ISSUE THREAD

**Issue by xenoflot** [WayofTime/BloodMagic#476]
tl;dr alchemical chemistry sets seem to cause a lag-inducing issue when the owner logs off.

---

Hi folks,

I play on a server with a custom pack. I'm building a base in an RFTools void. Everything is in a 3x3 chunk volume and I'm mindful to keep multiblock structures within a single chunk wherever possible. The Dimension Builder is in the overworld spawn chunks so always kept powered. Base is only loaded by a single ChickenChunks loader in the centre that's bound to my player.

I have a Tier 6 blood altar that hasn't given me any trouble.  I then set up about 16 alchemical chemistry sets with Ballads of Alchemy. Each chem set had it's own blood orb, bound to my player. When my player is online, no issues whatsoever but when I'd log off, the server would start lagging badly. The admins claimed that /cofh tps placed the issue firming on my void dim (and pulled the dim from the dim builder and shoved it in a chest).

---

To test, I logged in both my main account and my 2nd account, visited the void base with both players and ran the /chunkloaders command (not sure what adds that command, opis perhaps?) which gives opens a separate java window clearly showing in blue which chunks are loaded by MC and in green which are chunk-loaded by a block.

I can see a blue ~10 chunk radius around the two players in the void. I sent both players back to spawn. The /chunkloaders window clearly shows that a few second later, the only chunks loaded in the void are the 9 chunks forced by Player1's chunkloader. So far, so good. 

I logged out Player1, the player that owns the chunkloader and those ~16 blood orbs in the alchemical chem sets.   The /chunkloaders window now shows a curious activity not unlike Conways Game of life, as about 4 or 6 chunks near my void base start flickering in and out. Also I can instantly notice the hit to server performance.

I logged Player1 back in and it stopped misbehaving. I broke all the altars on top of the master ritual stones and re-tested with the same result.

I then broke all the ritual stones and alchemical sets, leaving just a sad looking collection of chests and AE2 connectors. Logged player1 out again and the void dim was completely unloaded, no server lag.

Conclusion: something in either the ballad of alchemy, the alchemical sets or the blood orbs seems to be causing lag. Perhaps it's having a futile search for it's bound owner?

Interestingly, I didn't touch the Tier 6 altar a couple of floors above. It has a blood orb sitting in it most of the time (SFM automated) but doesn't cause any such issues.

If I can provide any more info, please let me know.

forge 10.13.4.1492
bloodmagic 1.7.10-1.3.3-13
mcjtylib 1.4.0
rftools 3.35

**Comment by TehNut** [comment]
Because the 1.7.10 build of BM is no longer being worked on, we are clearing out non-critical issues for that version. Please let us know if the issue is persistent in the 1.8 releases.
